src/ophys_etl/transforms/trace_transforms.py

```diff
@@ -57,3 +63,109 @@ def extract_traces(movie_frames: np.ndarray, rois: List[coo_matrix],
                 traces[indx, time_slice] = raw_trace
 
     return traces
+
+
+def robust_std(x: np.ndarray) -> float:
+    """Compute the median absolute deviation assuming normally
+    distributed data. This is a robust statistic.
+
+    Parameters
+    ----------
+    x: np.ndarray
+        A numeric, 1d numpy array
+    Returns
+    -------
+    float:
+        A robust estimation of standard deviation.
+    Notes
+    -----
+    If `x` is an empty array or contains any NaNs, will return NaN.
+    """
+    mad = np.median(np.abs(x - np.median(x)))
+    return 1.4826*mad
+
+
+def noise_std(x: np.ndarray, filter_length: int = 31) -> float:
+    """Compute a robust estimation of the standard deviation of the
+    noise in a signal `x`. The noise is left after subtracting
+    a rolling median filter value from the signal. Outliers are removed
+    in 2 stages to make the estimation robust.
+
+    Parameters
+    ----------
+    x: np.ndarray
+        1d array of signal (perhaps with noise)
+    filter_length: int (default=31)
+        Length of the median filter to compute a rolling baseline,
+        which is subtracted from the signal `x`. Must be an odd number.
+
+    Returns
+    -------
+    float:
+        A robust estimation of the standard deviation of the noise.
+        If any valurs of `x` are NaN, returns NaN.
+    """
+    if any(np.isnan(x)):
+        return np.NaN
+    noise = x - medfilt(x, filter_length)
+    # first pass removing positive outlier peaks
+    # TODO: Confirm with scientific team that this is really what they want
+    # (method is fragile if possibly have 0 as min)
+    filtered_noise_0 = noise[noise < (1.5 * np.abs(noise.min()))]
```

Comment:
defaulted arg for 1.5? at least then it is documented.

Reply:
I did this at first, but I don't know if it's too confusing. I am not sure there is a reason for it to be configurable (as with your comment below)... It would have to be propagated all the way up to the base schema. @ledochowitsch what do you think?

Reply:
I don't think it needs to propagate all the way up to the user. Just define it with a default value and say: `noise below X threshold is filtered` or something. I find it useful to have such things described as parameters in the function docstrings.

Reply:
If it's just for documentation purposes should I just add more description? I am not sure the increase in parameter complexity is warranted if we're not actually making it available to the user.

Reply:
as you like